### PR TITLE
Changed Caption 

### LINKED ISSUE #7: Parking Lot
## Post any questions you may have here during the training.

### PR DESCRIPTION
**Pull request by preston-ttco**
## IT'S AN UPDATE, FOLKS!!!! 

|  | Updates|
|-----:|-----------|
|     1| New hat |
|     2| New socks  |

This closes #7 

<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/25423296/163456776-7f95b81a-f1ed-45f7-b7ab-8fa810d529fa.png">
  <source media="(prefers-color-scheme: light)" srcset="https://user-images.githubusercontent.com/25423296/163456779-a8556205-d0a5-45e2-ac17-42d089e3c3f8.png">
  Shows an illustrated sun in light mode and a moon with stars in dark mode.
</picture>

### CODE REVIEW

File: _slides/05-preston-ttco.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
```
Comment (preston-ttco) [inline _slides/05-preston-ttco.md:6]: what are you thinking dude

**Review by preston-ttco — COMMENTED** [review]
I APPROVE

File: _slides/05-preston-ttco.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
+HELLO!!!!!
```
Comment (NT001487) [inline _slides/05-preston-ttco.md:6]: I don't like this. change it

**Review by NT001487 — CHANGES_REQUESTED** [review]
Do something better.

**Review by NT001487 — APPROVED** [review]
changed my mind. I love it

**Pull request merged**